.eslintrc

```diff
@@ -23,8 +23,6 @@
     "node": true
   },
   "rules": {
-    "comma-dangle": ["error", "never"],
-    "semi": ["error", "never"],
```

Comment:
prettier does disable these rules as a precautionary measure - but there is an advantage to having them enabled (as long as you define them in a way that matches the prettier config) which is that it enables linting tools (eg atom eslint plugin) to pick up and highlight any issues. Without the eslint rule enabled we only get auto-formatting from prettier, and no linting from eslint.

Reply:
Interesting, I'm using `linter-eslint` on atom which lints for mismatches between the prettier output and the actual code via the `prettier/prettier` rules. Is that an option for you?

The concern with having rules present which are automatically disabled by `eslint-config-prettier` is that they are effect the rules are potentially misleading "dead code", given that `.prettierrc` governs these. Better to have a single declaration, in the spirit of DRY.

Reply:
Here's an example of that output btw: https://travis-ci.org/LN-Zap/zap-desktop/jobs/395132416

Reply:
Actually, I just tested this in atom and it does indeed highlight errors with the `semi` and `comma-dangle` issues via the `prettier/prettier` rules when the rules are not directly present in `.eslintrc`. Maybe I had some caching issue when I was testing previously.

Given that, I think this one is good to go.

unACK https://github.com/LN-Zap/zap-desktop/pull/460/commits/3c08966fd98d9ae26ec397ff73b70a48a80c4ea0